src/vuestic-theme/vuestic-components/va-input/VaInput.demo.vue

```diff
@@ -128,12 +128,40 @@
         :error-messages="['one', 'two']"
       />
     </VbCard>
-    <VbCard title="Textarea">
+    <VbCard title="Textarea (autosize: false)">
```

Comment:
My take on naming. Similarly descriptive, but easier to read.

* "Textarea" for deault case.
* "Textarea limits"
* "Textarea autogrow"
* "Textarea autogrow limits"